i18n/fr.yaml

```diff
@@ -1,83 +1,71 @@
 view_on_github:
-  other: "View on GitHub"
+  other: "Voir sur GitHub"
 contact_us:
-  other: "Contact us"
+  other: "Nous contacter"
 paragraph_1:
-  other: 'COVID Shield is a private, secure, and easy-to-use tool to help
-    governments launch their own exposure notification systems. It is
-    based on the exposure notification technology <a
-    href="https://www.apple.com/covid19/contacttracing">provided by Apple
-    and Google</a>, the most privacy-preserving approach currently
-    available.'
+  other: 'COVID Shield est un outil privé, sécurisé et facile à utiliser, qui vise à aider les gouvernements à lancer leurs propres systèmes de notification d'exposition. L'outil est basé sur une technologie de notification d'exposition <a
+    href="https://www.apple.com/covid19/contacttracing">fournie par Apple et Google</a>, dont l'approche scrupuleuse en matière de confidentialité n'a pas d'équivalent sur le marché actuel.'
 paragraph_2:
-  other: "For exposure notification to be most effective, it is important that
-    governments launch applications that can work together. COVID Shield
-    is flexible enough to be deployed at various levels of government by
-    enabling the server and mobile app to talk to each other even if there
-    are multiple instances."
+  other: "Pour que l'efficacité du système de notification d'exposition soit optimale, il est important que les gouvernements lancent des applications compatibles les unes avec les autres. COVID Shield est suffisamment flexible pour être déployé à différents ordres de gouvernement en activant le serveur et l'application mobile de manière qu'ils communiquent même en présence de plusieurs instances."
 related_reading_title:
-  other: "Related reading"
+  other: "Articles connexes"
 related_reading_1:
-  other: "How does Exposure Notification work?"
+  other: "Comment fonctionne la notification d'exposition ?"
 related_reading_2:
-  other: "Why is Exposure Notification important?"
+  other: "Pourquoi la notification d'exposition est-elle importante ?"
 related_reading_3:
-  other: "Why use Apple and Google’s technology?"
+  other: "Pourquoi utiliser la technologie d'Apple et de Google ?"
 privacy_title:
-  other: "Privacy is our top priority"
+  other: "La confidentialité est notre priorité absolue"
 privacy_content:
-  other: "<strong>COVID Shield takes a privacy-first approach to exposure notification.</strong> The COVID Shield app, its server, and the web-based results portal are all designed to preserve personal privacy while helping people safely return to work."
+  other: "<strong>L'approche de la notification d'exposition adoptée par COVID Shield place la confidentialité au rang de priorité.</strong> L'application COVID Shield, son serveur et le portail web de résultats sont tous trois conçus pour protéger la vie privée tout en aidant les gens à reprendre le travail en toute sécurité."
```

Comment:
"web" should have a capital "w", so this should read:
`<strong>L'approche de la notification d'exposition adoptée par COVID Shield place la confidentialité au rang de priorité.</strong> L'application COVID Shield, son serveur et le portail Web de résultats sont tous trois conçus pour protéger la vie privée tout en aidant les gens à reprendre le travail en toute sécurité.`